main/main.cpp

```diff
@@ -510,6 +510,7 @@ void Main::print_help(const char *p_binary) {
 	OS::get_singleton()->print("                                    The target directory must exist.\n");
 	OS::get_singleton()->print("  --export-debug <preset> <path>    Export the project in debug mode using the given preset and output path. See --export-release description for other considerations.\n");
 	OS::get_singleton()->print("  --export-pack <preset> <path>     Export the project data only using the given preset and output path. The <path> extension determines whether it will be in PCK or ZIP format.\n");
+	OS::get_singleton()->print("  --export-files <file1,file2,...>  Specific files to export. It overrides the 'Resources to export' option of preset.\n");
```

Comment:
This should probably be made more clear, making it clear that the files have to be one uninterrupted string with no spaces

Reply:
> Specific files to export.A string of filenames separated by comma(no spacees interrupted). It overrides the 'Resources to export' option of preset.

Do you think this is okay?

Reply:
Sounds good I think, though the suggestion on the proposal was to allow files and directories, and to have separated by spaces, would need to be discussed 🙂

Reply:
I just override the  the presets.selected_files. Didn't modified anything else. So this --export-files indeed  supports both directory path and file path.
![image](https://github.com/godotengine/godot/assets/1620585/340e6df3-2c53-4395-a28a-5e8b13013a73)

Or I can just use path not file to confuse people.
> --export-paths <path1,path2,...>  Specific paths(starts  with res://) to export. A string of paths separated by comma(no spacees interrupted). It overrides the 'Resources to export' option of preset.\n"

Reply:
But the files are files only, not directories, many places check specifically for files

Reply:
The proposal  I write does not talk about support both file and directory, just file.
But It exactly a problem that I didn't check if path is file or directory.
I think we should make it pure. `--export-files` just files.

Reply:
I'm talking about this: https://github.com/godotengine/godot-proposals/issues/8748#issuecomment-1873329157

I think just files is a bit too restrictive, but it'll be something to discuss

Reply:
Yeb. It's not hard to support directory path. Just find out the direcotry path string and recurse it. I will take a try.

Reply:
I did it. Add support for both file and directory and test it locally.

The cli help I change to this.
```shell
--export-files <path1,path2,...>  Specific a comma-separated list of files and folders to export(with 'res://'). It overrides the 'Resources to export' option of preset.
                                    WARN: It must be used with --export-release\debug\pack.It doesn't export but just override preset. 
```

Reply:
And the 'expot files without preset' is too difficult for me. We can't export without preset as it targets platforms. I don't know if it is good to create default preset to insteal `p_preset`. That is export workflow thing. I'm noob I can't handle this. 
So this PR just contains --export-files argument.